lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -166,10 +166,17 @@
                         courseID = this.courseID;
                     self.getTopic(topicID).done(function(topic) {
                         self.getTeam(teamID).done(function(team) {
-                            var view = new TeamProfileView({
-                                courseID: courseID,
-                                model: team
-                            });
+                            var readOnly = !(
+                                self.$el.data('privileged') ||
+                                    _.any(team.attributes.membership, function (membership) {
+                                        return membership.user.id === self.$el.data('username');
```

Comment:
I am confused why we are comparing user.id to username. Looking at https://openedx.atlassian.net/wiki/display/TNL/Team+API, it does appear that the fields should be username instead of id.